site/guide/manage-inventory-custom-fields.qmd

```diff
@@ -0,0 +1,66 @@
+---
+title: "Manage model inventory custom fields"
+date: last-modified
+---
+
+Create and edit custom fields that appear on all models in your model inventory. 
+
+## Prerequisites
+
+To configure templates, you must hold the `Customer Admin` role. 
+
+## Add or edit custom fields
+
+1. [Log in to the ValidMind UI]({{< var vm_url >}}).
+
+2. Head to your [**Settings**]({{< var vm_url >}}/settings).
+
+3. Under Workspace Settings, click on **Custom Fields**. Here you can edit existing custom fields, or add a new one:
+
+    - To add a new field, click **+ Add new field**. 
+    - To edit an existing field, select **{{< fa pencil >}} Edit** under the Actions column for the desired field.
+
+    At a minimum, you must provide the required information indicated by `*`. You can also choose to indicate that a field is **Required on registration**.
+
+4. When you are satisfied with the setup of your field, click **Save**.
+
+::: {.callout-tip appearance="minimal"}
+See the below table for information on available custom field types. 
+:::
+
+#### Custom field types
+
+| Field type | Details | 
+|---:|---|
+| Single Line Text | Simple text value. |
+| Long Text | Toggle **Enable rich text formatting** to create a template using the rich text editor. |
+| Single Select | Click **+ Add Item** to define a list of options. |
+| Multiple Select | Click **+ Add Item** to define a list of options. |
+| Checkbox | A `true`/`false` value set by a toggle. |
+| Number | Text value in valid number format. |
+| URL | Text value in valid URL format. |
+| Date | Date value in `yyyy-mm-dd` format. |
+| Date Time | Date value in `yyyy-mm-dd, 24hr` format. |
+| Email |Text value in valid email (`user@domain.com`) format. |
+| User | Select list pre-populated with users from your [User Directory]({{< var vm_url >}}/settings/user-directory). |
+: {.hover tbl-colwidths="[25,75]"}
+
+## Delete custom fields
+
+1. [Log in to the ValidMind UI]({{< var vm_url >}}).
+
+2. Head to your [**Settings**]({{< var vm_url >}}/settings).
+
+3. Under Workspace Settings, click on **Custom Fields**. 
+
+4. Select **{{< fa pencil >}} Edit** under the Actions column for the field you'd like to delete.
+
+5. Click **Delete**.
+
+    A confirmation box will appear; either cancel or verify removal of the field.
```

Comment:
```suggestion
    After you confirm the deletion, the field is removed.
```

Rather than explaining the behavior of the UI, explain the user action.